libs/utils/process_stub requires either MinGW or <sys/ptrace.h> on *NIX.

--- origsrc/qt-creator-2.8.1-src/src/libs/libs.pro	2013-08-23 05:27:18.000000000 -0500
+++ src/qt-creator-2.8.1-src/src/libs/libs.pro	2013-09-10 17:59:08.811495900 -0500
@@ -21,7 +21,6 @@ for(l, SUBDIRS) {
 }
 
 SUBDIRS += \
-    utils/process_stub.pro \
     qtcomponents/styleitem
 
 exists(../shared/qbs/qbs.pro):SUBDIRS += \
@@ -29,6 +28,7 @@ exists(../shared/qbs/qbs.pro):SUBDIRS +=
     ../shared/qbs/src/plugins \
     ../shared/qbs/static.pro
 
+!cygwin-*:SUBDIRS += utils/process_stub.pro
 win32:SUBDIRS += utils/process_ctrlc_stub.pro
 
 # Windows: Compile Qt Creator CDB extension if Debugging tools can be detected.    
